### PR TITLE
Feature popup window

### PR DESCRIPTION
**Pull request by darikmohammed**
# Finish up Detail Popup Window
> Implement a popup window that includes the project details in both desktop and mobile.

- When the project button is clicked the popup with details about the project appears.
- When the close (X) button is clicked the popup disappears. 
- The popup implemented for both mobile and desktop view. 
- Used Javascript array to store the information of the projects. 
- The project section is created dynamically. 

https://user-images.githubusercontent.com/56404835/167925670-573789cc-07f3-4871-9760-c8b3eb6b8e89.mov

https://user-images.githubusercontent.com/56404835/167926007-9b25cf38-602f-4541-9832-29687df986c1.mov

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,78 +76,45 @@ <h2 class="primary-color supporting-headline">
...
-              <button type="button" class="btn">See Project</button>
-            </div>
+          <div class="project-btn mobile-view-btn">
+            <a href="" class="moblie-see-live-btn btn">
+              See live
+              <img src="./icons/SeeProjectIcon.svg" alt="See live" />
+            </a>
+            <a href="" class="moblie-see-source-btn btn">
+              See Source <img src="./icons/Vector.svg" alt="" />
+            </a>
```
Comment (sja-thedude) [inline index.html:117]: - [ ] Kindly add the section of the entire work dynamically from js by removing the existing code from HTML and creating it dynamically from js.

File: style.css [hunk 2]
```diff
@@ -258,10 +262,92 @@ p {
...
+}
+
+.modal .project-img {
+  align-self: center;
+  display: flex;
+  justify-content: center;
+}
+
+.modal .project-img img {
+  width: 100%;
```
Comment (sja-thedude) [inline style.css:305]: Desktop popup:

- [ ] Kindly make sure to follow the height and width for the popup as per the Figma design.
- [ ] Kindly make sure that when you click on `see project `and a popup appears, the background of the page is blurred and follows the colour displayed as per the Figma design.
- [ ] Kindly make sure you have the correct spacing to the top, right, left and bottom of the popup as per the Figma design.
- [ ] Kindly make sure to follow the correct size for buttons as per the Figma design.
- [ ] Kindly reduce the spacing between the title of the popup and technologies as per the Figma design.
- [ ] Kindly follow the font size and spacing for the popup paragraph, and title.

Your design:
Screenshot 2022-05-11 at 10 34 56 PM


Figma design:
Screenshot 2022-05-11 at 10 30 24 PM

File: style.css [hunk 3]
```diff
@@ -258,10 +262,92 @@ p {
...
+  height: 220px;
+}
+
+.project-intro .project-title h2 {
+  font-weight: 600;
+  font-size: 2rem;
+  color: #172b4d;
+}
```
Comment (sja-thedude) [inline style.css:313]: Mobile popup:

- [ ] Kindly make sure that when you click on `see project `and a popup appears, the background of the page is blurred and follows the colour displayed as per the Figma design.
- [ ] Kindly make sure you have the correct spacing to the top, right, left and bottom of the popup and all popup items as per the Figma design.
- [ ] Kindly make sure to follow the correct size for buttons and also place them in the centre as per the Figma design.
- [ ] Kindly reduce the spacing between the title of the popup and technologies as per the Figma design.
- [ ] Kindly remove extra technologies as per Figma design.
- [ ] Kindly make sure the popup is displayed in the centre with equal margins. As you can see there is more margin on the right. Fix it.

Your design:
Screenshot 2022-05-11 at 10 35 22 PM


Figma design:
Screenshot 2022-05-11 at 10 30 51 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @darikmohammed, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job so far in implementing the popup.
✔️ All linters are passing
✔️ Correct git-flow
✔️ Good PR Title and Description
✔️ Professional Readme

## Required 
- [ ] Kindly add collaborator details to your readme.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (darikmohammed) [reply index.html:117]: @sja-thedude I have removed all the HTML code for the entire work section and created it dynamically from the js file. Kindly Please check it(on the commit e0ab089f7c51512028f60c32de5d185b47653a84 or you can see it in the index.js file from line number 120 to 141 ). But I have created the popup window using my HTML and appended the necessary data from the js file dynamically. Is any problem with that?

**Review by gemmen29 — APPROVED** [review]
Hi @darikmohammed, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**